src/hooks/useTimer.ts

```diff
@@ -100,10 +100,42 @@ function useTimer() {
     setIsRunning(true)
   }
 
+  const resumePast = async (jobId: string, task: string, timeLogId: string) => {
+    if (isRunning) {
+      if (currentTimelog?.timelogId === timeLogId) {
+        await pauseResumeTimerMutation.mutateAsync({
+          timelogId: timeLogId,
+          timer: 'stop',
+        })
+
+        setIsRunning(false)
+      }
+
+      return
+    }
+
+    const job = jobs.find((j) => j.jobId === jobId)
+
+    setCurrentTimelog({
+      jobId,
+      jobName: job!.jobName,
+      timelogId: timeLogId,
+      taskName: task,
+    })
+
+    await pauseResumeTimerMutation.mutateAsync({
+      timelogId: timeLogId,
+      timer: 'start',
+    })
+
```

Comment:
I think this fails when the timelog is on a day in the past, so you might need to check for that and start a new timer.

Reply:
Did you verify this? I think you cannot restart a timelog if it is in the past.

Reply:
I added it to the button that runs this action, but now I also moved it to the function